src/Microsoft.AspNetCore.Mvc.Core/Filters/MiddlewareFilterAttribute.cs

```diff
@@ -0,0 +1,43 @@
+﻿// Copyright (c) .NET Foundation. All rights reserved.
+// Licensed under the Apache License, Version 2.0. See License.txt in the project root for license information.
+
+using System;
+using Microsoft.AspNetCore.Mvc.Filters;
+using Microsoft.AspNetCore.Mvc.Internal;
+using Microsoft.Extensions.DependencyInjection;
+
+namespace Microsoft.AspNetCore.Mvc.Core.Filters
```

Comment:
This should be in `Microsoft.AspNetCore.Mvc` - .Filters is for filter-related contracts. Stuff that will appear prominently in user code should be in `.Mvc`